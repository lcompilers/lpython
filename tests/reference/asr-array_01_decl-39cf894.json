{
    "basename": "asr-array_01_decl-39cf894",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/array_01_decl.py",
    "infile_hash": "e5f2b6082b67a8a7847e7051d7b4d3c954d1ea88d865326810543dca",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-array_01_decl-39cf894.stderr",
    "stderr_hash": "1faef2a94bfd57fd06681e9168a3089ce3a5e56f7c93f02a8d4f48ec",
    "returncode": -2
}