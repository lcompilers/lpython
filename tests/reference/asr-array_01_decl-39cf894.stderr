KeyboardInterrupt
The command 'python $DIR/src/bin/../runtime/lpython_parser.py tests/../integration_tests/array_01_decl.py tests/../integration_tests/array_01_decl.py9DCD451' failed.
